ai_edge_torch/generative/examples/stable_diffusion/pipeline.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2024 The AI Edge Torch Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import os
+from pathlib import Path
+from typing import Dict, Optional
+
+import numpy as np
+from PIL import Image
+from tqdm import tqdm
+
+import ai_edge_torch.generative.examples.stable_diffusion.samplers as samplers
+from ai_edge_torch.generative.examples.stable_diffusion.tokenizer import Tokenizer  # NOQA
+import ai_edge_torch.generative.examples.stable_diffusion.util as util
+from ai_edge_torch.model import TfLiteModel
+
+
+class StableDiffusion:
+
+  def __init__(
+      self,
+      *,
```

Comment:
what's this for?

Reply:
This requires the following parameters to be specified by key words.